### ISSUE TITLE
Current CRA install is incompatible with 1.0

### ISSUE THREAD

**Issue by with-heart** [chakra-ui/chakra-ui#956]
I wanted to open this to make sure we're tracking this issue and handle it through documentation in some way. As noted in #909, the current version of CRA targets `typescript: ~3.7.2`, which is not recent enough to be compatible with Chakra (which takes advantage of the `import type`/`export type` syntax from 3.8).

**Comment by aaronmcadam** [comment]
Did we implement the CRA support with a template or a fork? Can templates add their own dependencies so that we could set the `typescript` version?

**Comment by mrmckeb** [comment]
Hi @with-heart, I can tell you that we're very close to releasing a new version of CRA, so that should solve the issue. 

In the meantime, CRA users can definitely bump their TS version with no issue (although they might find that the included ESLint config doesn't support the new syntax).

**Comment by with-heart** [comment]
@aaronmcadam Not yet! I have a draft PR in #936 that I need to need to write a guide for before merging, so it's close. Supported ts version is in the ts template already :)

@mrmckeb Awesome! Thanks for letting me know. I'll be sure to add a note about the ESLint config to the template guide until the new version is published.

**Comment by mrmckeb** [comment]
This is being held up by a last-minute CRA change to service workers, and we hope to have this out in the coming days... but can't promise. Sorry all.

**Comment by with-heart** [comment]
@mrmckeb It's all good! We're held up a little as well trying to get through issues, so it's no big deal :) Thanks for keeping us posted!

**Comment by with-heart** [comment]
Gonna close this as a duplicate of #913
